card-bot/.vscode/launch.json

```diff
@@ -0,0 +1,65 @@
+{
+    "configurations": [
+
+        {
+            "type": "aws-sam",
+            "request": "direct-invoke",
+            "name": "cardBot:getFirstCard.lambdaHandler (nodejs12.x)",
+            "invokeTarget": {
+                "target": "code",
+                "projectRoot": "",
+                "lambdaHandler": "getFirstCard.lambdaHandler"
+            },
+            "lambda": {
+                "runtime": "nodejs12.x",
+                "payload": {},
+                "environmentVariables": {}
+            }
+        },
+        {
+            "type": "aws-sam",
+            "request": "direct-invoke",
+            "name": "cardBot:getNextCard.lambdaHandler (nodejs12.x)",
+            "invokeTarget": {
+                "target": "code",
+                "projectRoot": "",
+                "lambdaHandler": "getNextCard.lambdaHandler"
+            },
+            "lambda": {
+                "runtime": "nodejs12.x",
+                "payload": {},
```

Comment:
@renjanlookose please update the payload required to run this

Reply:
ok .. will do